projects/wireshark/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Wireshark build.sh script inspired from projects/gnutls/build.sh by Alex Gaynor.
+
+FUZZ_DISSECTORS="ip \
+  udp"
+
+export WIRESHARK_LIBS_PATH="$SRC/libs"
+mkdir -p "$WIRESHARK_LIBS_PATH"
+
+# compile static version of libs
+# XXX, with static wireshark linking each fuzzer binary is ~238 MB (just libwireshark.a is 423 MBs).
```

Comment:
looking at other projects, it seems that static linking is preferred, is it also for wireshark if single binary is 238 MB?